mathml/relations/css-styling/mathsize-attribute.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8"/>
+    <title>Verify mathsize attribute</title>
+    <link rel="help" href="https://mathml-refresh.github.io/mathml-core/#legacy-mathml-style-attributes">
+    <meta name="assert" content="Verify mathsize attribute values.">
+    <link rel="match" href="mathsize-attribute-ref.html">
+  </head>
+  <body>
+
+    <!-- This verifies the effect of the mathsize attribute. -->
+    <div>
+      <math>
+        <mi mathsize="small">x</mi>
+        <mi mathsize="normal">x</mi>
+        <mi mathsize="big">x</mi>
+        <mi mathsize="2em">x</mi>
+        <mi mathsize="2">x</mi>
```

Comment:
small, normall, big and unitless have been removed from MathML core. So we only need to test <lengh-percentage> values. I guess this is taken from WebKit so I imagine the values should be still checked with the MathMLCore option disabled, but moved to WebKit's LayoutTests.

Reply:
"medium" and "2" have been remove too. The def is https://mathml-refresh.github.io/mathml-core/#attribute-mathsize